src/main/webapp/userAuthScript.js

```diff
@@ -32,8 +33,38 @@ function handleResponse(response) {
 function addDialogueToDom(authInfo) {
   const quoteContainer = document.getElementById(LOGIN_CONTAINER);
   if (authInfo.includes("Logout")) {
-    window.location.replace(window.location.origin + "/characterDesign.html");
-  } else {
-    quoteContainer.innerHTML = authInfo;
+    loadOptions(authInfo);
   }
+  quoteContainer.innerHTML = authInfo;
+  
+}
+
+function loadOptions (logoutLink) {
+  let lastStageButton = document.createElement("button");
+  let newGameButton = document.createElement("button");
+  let lastStageLink = document.createElement("a");
+  let newGameLink = document.createElement("a");
+
+  lastStageButton.className = "step-button";
+  newGameButton.className = "step-button";
+  lastStageButton.innerText = "Return to Last Stage";
+  newGameButton.innerText = "New Game";
+
+  lastStageLink.href ="gameStage.html";
+  lastStageLink.appendChild(lastStageButton);
+  newGameLink.href = "characterDesign.html";
+  newGameLink.appendChild(newGameButton);
```

Comment:
To add more readability, you could refactor into a createButtonWithLink method, i.e.
```
function createButtonWithLink(text, className, linkHref) {
  let button = document.createElement('button');
  button.innerText = text;
  button.className = className;
  let link = document.createElement('a');
  link.href = linkHref;
  link.appendChild(button);
  return link;
}
```
and then create lastStageLink and newGameLink through 2 function calls. Up to you though!